core/src/main/java/nl/nn/adapterframework/configuration/IbisContext.java

```diff
@@ -351,6 +351,7 @@ private void digestClassLoaderConfiguration(ClassLoader classLoader,
 						databaseMigrator.close();
 					}
 					catch (Exception e) {
+						ConfigurationWarnings.add(LOG, e.getMessage());
```

Comment:
I don't think it is a good idea to log this message in the ConfigurationWarnings as well as in the MessageKeeper.

Reply:
I do agree